src/main/java/com/deavensoft/paketomat/center/CenterController.java

```diff
@@ -65,6 +69,7 @@ public int savePackage(@RequestBody Package newPackage)
             emailDetails.setSubject("test");
             model.put("msgBody", emailDetails.getMsgBody());
             emailService.sendMailWithTemplate(emailDetails, model);
+            dispatcherService.delieverPackage(newPackage);
```

Comment:
Unreported exception NoSuchUserException; must be caught or declared to be thrown.

Reply:
We added exception to throws clause

Reply:
If you have NoSuchUserException, please throw it.

Reply:
Done